### DISCUSSION TITLE
Picketfence analysis keeps stating list index out of range

### DISCUSSION THREAD

**Discussion by JJOtten** [jrkerns/pylinac#437] [Q&A]
Hi.

I'm still relatively new to pylinac. I've been trying to analyse to picketfences but keep getting an error that states "list index out of range". Not sure how to adjust the values such that the script analyses the film? 

Github doesn't support DICOM files so here is a link to the files: https://drive.google.com/drive/folders/1fq0okFz0x2nRotslo10wFQFfc5-wqWf2?usp=sharing

I appreciate any help

**Comment by jrkerns** [comment]
The pickets you're using appear to be nearly abutting. Pylinac cannot determine the FWHM of the pickets clearly when they abut so closely. See here on tips to get good images: https://pylinac.readthedocs.io/en/latest/picketfence.html#acquiring-good-images and also the "Restrictions" section here: https://pylinac.readthedocs.io/en/latest/picketfence.html#algorithm
